doc/routinator.1

```diff
@@ -313,6 +313,12 @@ This option provides the URI of a proxy to use for all HTTP connections made
 by the RRDP client. It can be either an HTTP or a SOCKS URI. The option can
 be given multiple times in which case proxies are tried in the given order.
 
+.TP
+.BI --max-object-size= BYTES
+Limits the size of individual objects received via either rsync or RRDP to
+the given number of bytes. The default value if this option is not present
+is 20 MBytes. Use a value of 0 to disable the limit.
```

Comment:
I could see users getting confused by the '20', thinking this option takes Megabytes instead of bytes and perhaps misconfigure their setup into a breaking way.
```suggestion
is 20000000 (i.e. 20 MBytes). Use a value of 0 to disable the limit.
```